storm-server/src/main/java/org/apache/storm/DaemonConfig.java

```diff
@@ -316,6 +316,12 @@
     @isBoolean
     public static final String UI_DISABLE_HTTP_BINDING = "ui.disable.http.binding";
 
+    /**
+     * This controls whether Storm UI would not monitor Spout lag.
```

Comment:
Nit: If I didn't already know about storm-kafka-monitor, this would be unclear to me. I would also be confused why this doesn't do anything for non-Kafka spouts. How about "This controls whether Storm UI displays spout lag for the Kafka spout"

Reply:
Addressed.